src/sentry/conf/server.py

```diff
@@ -776,6 +776,12 @@ def create_partitioned_queues(name):
 # django-rest-framework
 
 REST_FRAMEWORK = {
+    "DEFAULT_RENDERER_CLASSES": ["rest_framework.renderers.JSONRenderer"],
+    "DEFAULT_PARSER_CLASSES": [
+        "rest_framework.parsers.JSONParser",
+        "rest_framework.parsers.MultiPartParser",
+        "rest_framework.parsers.FormParser",
```

Comment:
In the old code FormParser was before MultiPartParser. Does that matter?

Reply:
https://www.django-rest-framework.org/api-guide/parsers/#how-the-parser-is-determined

> The set of valid parsers for a view is always defined as a list of classes. When request.data is accessed, REST framework will examine the Content-Type header on the incoming request, and determine which parser to use to parse the request content.

Doesn't look like the list is interpreted in-order.